src/xpk/commands/workload.py

```diff
@@ -521,6 +538,7 @@ def workload_create(args) -> None:
         storage_annotations=storage_annotations,
         service_account=service_account,
     )
+  print(yml_string)
```

Comment:
Is this a leftover from some debugging? Should we use xpk_print here? The yaml can be quite big do we really want to write with each execution?